src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -820,6 +835,112 @@ CoordinatedRemoteTransactionsAbort(void)
 
 
 /*
+ * CoordinatedRemoteTransactionsSavepointBegin sends the SAVEPOINT command for
+ * the given sub-transaction id to all connections participating in the current
+ * transaction.
+ */
+void
+CoordinatedRemoteTransactionsSavepointBegin(SubTransactionId subId)
```

Comment:
Naming-wise I think this should be `RemoteSavepointBegin` or maybe `RemoteTransactionSavepointBegin`, rather than the current name. The worker transaction stuff doesn't have the Coordinated* prefix.